internal/controller/driver_controller.go

```diff
@@ -504,6 +567,13 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 											utils.CsiAddonsEndpointContainerArg,
 											"",
 										),
+										utils.If(logRotationEnabled, utils.LogToStdErrContainerArg, ""),
+										utils.If(logRotationEnabled, utils.AlsoLogToStdErrContainerArg, ""),
+										utils.If(
+											logRotationEnabled,
+											utils.LogFileContainerArg(fmt.Sprintf("csi-%splugin", r.driverType)),
```

Comment:
what will happen if the controller-plugin and the node-plugin run on the same node? are they going to log into different files? i don't think so but need confirmation on this one.

Reply:
FIle name would be same, but underlyning directory would be different ass per the volume,
`utils.LogsDirVolume(logHostPath, daemonSet.Name)`
`utils.LogsDirVolume(logHostPath, deploy.Name),`